docs/dusty-specs/app-specs.md

```diff
@@ -0,0 +1,132 @@
+# Dusty App Specs
+These specs are used to specify apps for you local environment. They should go in the apps folder in your specs repo.
+
+## Keys
+
+### repo
+```
+repo: github.com/gamechanger/app1
+repo: /Users/dusty/app1
+```
+This repo key is used to specify a local or remote github repo where the code for the app lives.
+
+### mount
+```
+mount: /gc/app1
+```
+The mount key is used to specify where in the container's file system to mount the code specified by the repo key.  The mounting is done using docker's volume mounts. <br />
```

Comment:
two new lines looks dumb when you are separating two single lines.  I single new line seems better to me.